SimplyPluralApi.yml

```diff
@@ -1,117 +1,221 @@
 openapi: '3.0.2'
 info:
   title: Simply Plural API
-  description: This is a work-in-progress API definition. If you need to know specifcs, please go to https://github.com/ApparyllisOrg/SimplyPluralApi/blob/main/src/api/v1/routes.ts
+  description: This is a work-in-progress API definition. If you need to know specifics, please go to https://github.com/ApparyllisOrg/SimplyPluralApi/blob/main/src/api/v1/routes.ts
   version: '1.0'
 servers:
   - url: https://api.apparyllis.com:8443
   - url: https://devapi.apparyllis.com
 
 paths:
-  /v1/member/{systemId}/{docId}:
+  /v1/members/{systemId}:
     get:
-      description: 'Get a member byId'
+      summary: 'Get All Members'
+      description: 'Get members from a system by system id. This can be the system of the authenticated user, or from a friend of the authenticated user.'
+      security:
+        - APITokenAuth: []
       tags:
         - Members
       parameters:
         - in: path
-          name: docId
+          name: systemId
           schema:
             type: string
           required: true
-          description: String Id of the member
+          description: String Id of the system (12 byte BSON ObjectId as string)
```

Comment:
That's not true actually, it's a Firebase User ID string format. However the API regex checks for a 
`^[A-z0-9]{0,50}$` match. It's not the length of a Firebase User ID, however there's no official length for a user id, and they will grow in the future without notice so we kept it large just in case they roll out an update and we're not aware of the change.